ibis/backends/tests/test_temporal.py

```diff
@@ -1337,6 +1337,13 @@ def test_interval_add_cast_column(backend, alltypes, df):
             ),
             "%Y%m%d",
             marks=[
+                pytest.mark.broken(
```

Comment:
`broken` should be used for marking tests that are failing for reasons unrelated to a given feature / PR (so a contributor isn't blocked by a random failure due to an upstream change).

If this operation will never be support by `datafusion` it should be a `pytest.mark.never`

Reply:
I'm leaving `strftime` out of the PR since is failing in some environments.